README.md

```diff
@@ -1,2 +1,57 @@
+```sh
+$ . ./set-env.sh
+```
+
+| Variable              | Value                            | Description                                                                  |
+|-----------------------|----------------------------------|----------------------------------------------------------------------------- |
+| PROV_APP_USER         | openshift                        | Provisioning app user name                                                   |
+| PROV_APP_PASSWORD     | openshift                        | Provisioning app password                                                    |
+| ATLASSIAN_USER        | openshift                        | Atlassian user name                                                          |
+| ATLASSIAN_PASSWORD    | openshift                        | Atlassian password                                                           |
+| JENKINS_USER          | developer                        | Jenkins user name                                                            |
+| JENKINS_PASSWORD      | any                              | Jenkins password                                                             |
+| OPENSHIFT_USER        | developer                        | Openshift user name                                                          |
+| OPENSHIFT_PASSWORD    | any                              | Openshift password                                                           |
+| PROV_APP_NAME         | openshift                        | Name of the deployment of the provisioning app                               |
+| JIRA_URL              | http://jira.odsbox.lan:8080/     | Url of Jira instance related with the prov app                               |
+| OPENSHIFT_PROJECT     | edpp                             | project identifier for prov app in the preliminary tests(jira tests)         |
+| OPENSHIFT_PUBLIC_HOST | ocp.odsbox.lan                   | host where we can locate the prov app                                        |
+| OPENSHIFT_CLUSTER     | https://ocp.odsbox.lan:8443/     | URL of the Openshift Cluster                                                 |
+| BITBUCKET_URL         | http://bitbucket.odsbox.lan:7990 | Url of Bitbucket instance                                                    |
+| SIMULATE              | false                            | Specify (true or false)  if we skip the creation of project, components, etc |
+
+The following commands will launch the tests with the individual browsers:
+
+```shell script
+# Headless tests, to use chrome without displaying the browser.
+$ ./gradlew chromeHeadlessTest --tests "org.ods.e2e.ODSSpec"
+
+
+# A test spec
+$ ./gradlew chromeTest --tests "org.ods.e2e.ODSSpec"
+# A specific test
+$ ./gradlew chromeTest --tests "org.ods.e2e.ODSSpec.FT_01_003"
+```
+
+Replace `./gradlew` with `gradlew.bat` in the above examples if you're on Windows.
+
+## Tests being implemented
+### Open Dev Stack
+```ODSSpec.groovy```
```

Comment:
@jorge-romero This should not be documented. The tests are the documentation.

Reply:
Ok, I'll remove that too